.github/workflows/dependency-graph.yml

```diff
@@ -0,0 +1,14 @@
+name: Update Dependency Graph
+on:
+  push:
+    branches:
+      - main # default branch of the project
+jobs:
+  dependency-graph:
+    name: Update Dependency Graph
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Should we use [v4 of `actions/checkout`](https://github.com/actions/checkout/blob/main/CHANGELOG.md)?

Reply:
Good spot, this was the workflow recommended on the [action's page](https://github.com/marketplace/actions/sbt-dependency-submission), so will try in a follow-up PR.